jwst/datamodels/schemas/multislit.schema.yaml

```diff
@@ -1,135 +1,10 @@
 allOf:
 - $ref: core.schema.yaml
-- $ref: bunit.schema.yaml
```

Comment:
Is the definition for BUNIT keywords included again in some other place? We need those in the SCI and ERR extension headers.